parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -390,6 +390,16 @@ ensureKubelet() {
     {{end}}
 }
 
+ensureLabels() {
+    KUBELET_DEFAULT_FILE=/etc/default/kubelet
```

Comment:
why do you need to wait on kubelet but ensureLabelNodes doesn't?

Reply:
This file is what stores the labels that we use as our env:
https://github.com/Azure/AgentBaker/blob/603788b85aa4502f05fcd2544d6d9dd89e47b150/parts/linux/cloud-init/artifacts/labels.service#L7